data/migrations/36.lua

```diff
@@ -1,3 +1,7 @@
 function onUpdateDatabase()
-	return false
+	print("> Updating database to version 37 (loyalty system)")
+	db.query("ALTER TABLE `accounts` ADD COLUMN `loyalty_points` SMALLINT UNSIGNED NOT NULL DEFAULT 0 AFTER `premium_ends_at`")
+	db.query("ALTER TABLE `players` ADD COLUMN `main_character` TINYINT(1) NOT NULL DEFAULT 0 AFTER `account_id`")
```

Comment:
I don't think this belongs in here, if anything it should be stored in the account. Even better if it uses account storage.

Another improvement would be storing premium time spans rather than a counter of premium days, so we can store _when_ a player was premium and not only _how long_

Reply:
Sure, I can move `main_character` column to account storages, I'm just using it as a way to choose which character I'd show on loyalty highscores.